tile.cpp

```diff
@@ -1841,15 +1874,19 @@ void preserve_attributes(std::map<std::string, attribute_op> const *attribute_ac
 	}
 }
 
-bool find_partial(std::vector<partial> &partials, serial_feature &sf, ssize_t &out, std::vector<std::vector<std::string>> *layer_unmaps, long long maxextent) {
+bool find_partial(std::vector<partial> &partials, serial_feature &sf, ssize_t &out, std::vector<std::vector<std::string>> *layer_unmaps, long long maxextent, ssize_t multiplier_seq) {
 	for (size_t i = partials.size(); i > 0; i--) {
 		if (partials[i - 1].t == sf.t) {
 			std::string &layername1 = (*layer_unmaps)[partials[i - 1].segment][partials[i - 1].layer];
 			std::string &layername2 = (*layer_unmaps)[sf.segment][sf.layer];
 
 			if (layername1 == layername2 && partials[i - 1].extent <= maxextent) {
-				out = i - 1;
-				return true;
+				if (multiplier_seq <= 0) {
```

Comment:
Can you add a comment explaining `find_partial`s role? I understand that it now returns true once every `[point multiplier]` times, but it's used in a bunch of places and I don't really understand the context

Reply:
Sure, will add a comment. It is used to find the feature that a dropped feature's attributes or geometry will be accumulated onto.